packaging/dbscripts/upgrade/04_05_0160_change_default_display.sql

```diff
@@ -0,0 +1,22 @@
+-- Change InstanceType's and Blank template default display type from QXL (1) to VGA (2)
```

Comment:
beside that corner case that you've mentioned in the description of the PR, this change is not specific to the 4.7 cluster level, can you please remind me if we talked about that?

Reply:
I don't remember if we talked about it. Looking on the patch and bug, seems as a general decision, regardless CL 4.7 (and so it seems based on the code and the last gap in it). Logically, both can fit. But, if we go with CL, we will need to think of many flows.